common/djangoapps/third_party_auth/api/views.py

```diff
@@ -389,3 +391,53 @@ def get_serializer_context(self):
         context['provider'] = self.provider
 
         return context
+
+
+class ThirdPartyAuthUserStatusView(APIView):
+    """
+    Provides an API endpoint for retrieving the linked status of the authenticated
+    user with respect to the third party auth providers configured in the system.
+    """
+    authentication_classes = (
+        JwtAuthentication, OAuth2AuthenticationAllowInactiveUser, SessionAuthenticationAllowInactiveUser
+    )
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request):
+        """
+        GET /api/third_party_auth/v0/providers/user_status/
+
+        **GET Response Values**
+        {
+            "accepts_logins": true,
```

Comment:
Can you explain what `accepts_logins` means in this context?

A docstring explaining each field, like in the [above View](https://github.com/edx/edx-platform/blob/712805992193da49e7ff72a0b396334c7494cac5/common/djangoapps/third_party_auth/api/views.py#L156-L171), would make this API more approachable.

Reply:
The is basically a straight copy from the existing template-rendering [view](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/settings_views.py#L150). I believe the template uses this value to determine whether or not it should display a "Login" button for the given provider on the account settings page.

Let's see how/if we end up using this on the new account settings page before fine-tuning the documentation.

Reply:
Ok. So do you plan to create a ticket to update the documentation for this API before calling this complete?

Reply:
The API work is done. We can follow up with a ticket later when the use of this API becomes solidified.